scalatest/src/main/scala/org/scalatest/tools/RunnerJFrame.scala

```diff
@@ -611,7 +611,7 @@ private[scalatest] class RunnerJFrame(
     exitSemaphore.acquire()
     addWindowListener(
       new WindowAdapter {
-        override def windowClosed(e: WindowEvent): Unit = { exitSemaphore.release() }
+        override def windowClosed(e: WindowEvent | Null): Unit = { exitSemaphore.release() }
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.